<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckpointsForm</class>
 <widget class="QWidget" name="CheckpointsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/png/resources/png/logo.png</normaloff>:/png/resources/png/logo.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>681</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Checkpoints</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Checkpoints</string>
     </property>
    </widget>
    <widget class="QLabel" name="infoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>711</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTreeWidget" name="chpsResults">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>701</width>
       <height>491</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sequence</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Index</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payload</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="clearBtn">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>550</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/png/resources/png/icons-bin.png</normaloff>:/png/resources/png/icons-bin.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="add_chp_btn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>114</width>
       <height>24</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add Sequence</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/png/icon-add</normaloff>:/png/icon-add</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="exportBtn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>550</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Export Checkpoints</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/png/resources/png/icons-save.png</normaloff>:/png/resources/png/icons-save.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="importBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>550</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Import Checkpoints</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/png/resources/png/icons-import.png</normaloff>:/png/resources/png/icons-import.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="verifyBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Verify</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/png/resources/png/icon-lens.png</normaloff>:/png/resources/png/icon-lens.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="addAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/png/icon-add</normaloff>:/png/icon-add</iconset>
   </property>
   <property name="text">
    <string>addChp</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
